pkg/sloop/queries/queryd3heatmap.go

```diff
@@ -63,11 +64,27 @@ func EventHeatMap3Query(params url.Values, t typed.Tables, queryStartTime time.T
 		return []byte{}, err
 	}
 
+	eventBucketWidthString := params.Get(BucketingParam)
+	if eventBucketWidthString == "" {
+		eventBucketWidthString = "1m"
+	}
+
+	eventBucketWidth, err := time.ParseDuration(eventBucketWidthString)
```

Comment:
what is the unite of eventBucketWidth here, should we force to be minutes, I noticed in the blow function we assume it is minute.